### PR TITLE
code changes for HomeFragment

### PR DESCRIPTION
**Pull request by LongNg89**
- #16 
- #20 
- Signed-off-by: LongNg89 <nghuy253@gmail.com>

### CODE REVIEW

File: app/src/main/java/com/longng/blog4pp/fragments/HomeFragment.java [hunk 1]
```diff
@@ -21,7 +44,94 @@ public HomeFragment() {
...
+                    }
+                }
+            });
+
+            if (mAuth.getCurrentUser() != null) {
+                //Order according to date
+                Query firstQuery = firebaseFirestore.collection("Posts")
+                        .orderBy("timeStamp", Query.Direction.DESCENDING)
+                        .limit(3);
+                firstQuery.addSnapshotListener(getActivity(), new EventListener<QuerySnapshot>() {
```
Comment (LongNg89) [inline app/src/main/java/com/longng/blog4pp/fragments/HomeFragment.java:79]: remove `getActivity()` in query

File: app/src/main/java/com/longng/blog4pp/fragments/HomeFragment.java [hunk 2]
```diff
@@ -21,7 +44,94 @@ public HomeFragment() {
...
+        return view;
+    }
+
+    private void loadMorePosts() {
+        if (mAuth.getCurrentUser() != null) {
+            Query nextQuery = firebaseFirestore.collection("Posts")
+                    .orderBy("timeStamp", Query.Direction.DESCENDING)
+                    .startAfter(lastVisible)
+                    .limit(3);
+            nextQuery.addSnapshotListener(getActivity(), new EventListener<QuerySnapshot>() {
```
Comment (LongNg89) [inline app/src/main/java/com/longng/blog4pp/fragments/HomeFragment.java:114]: remove `getActivity()` in query

**Review by LongNg89 — COMMENTED** [review]
@Bunrealbo

File: app/src/main/java/com/longng/blog4pp/fragments/HomeFragment.java (hunk 1 above)
Comment (Bunrealbo) [reply app/src/main/java/com/longng/blog4pp/fragments/HomeFragment.java:79]: I was fixed `error`

File: app/src/main/java/com/longng/blog4pp/fragments/HomeFragment.java (hunk 2 above)
Comment (Bunrealbo) [reply app/src/main/java/com/longng/blog4pp/fragments/HomeFragment.java:114]: I was fixed `error`

**Pull request closed without merging**